VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -140,16 +142,31 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
                 }
                 return participant;
             });
+        const quickLinkParticipants = this.participantsInConsultation
+            .filter(p => p.role === Role.QuickLinkParticipant)
```

Comment:
QuickLinkObserver ?

Reply:
Quick link observers are not to be displayed in these lists except in joh consultation rooms. They are included in getWitnessesAndObservers.